app/v1/support/DesResponseMappingSupport.scala

```diff
@@ -20,10 +20,26 @@ import utils.Logging
 import v1.controllers.EndpointLogContext
 import v1.models.errors._
 import v1.models.outcomes.ResponseWrapper
+import v1.models.response.retrieveBusinessDetails.{RetrieveBusinessDetailsResponse, RetrieveBusinessDetailsResponseList}
 
 trait DesResponseMappingSupport {
   self: Logging =>
 
+  final def filterId(
```

Comment:
You need to write tests for this function in test/v1/support/DesResponseMappingSupportSpec